### ISSUE TITLE
Configuring Java heap size for ontology loading

### ISSUE THREAD

**Issue by balhoff** [EBISPOT/ontotools-docker#7]
I'm encountering a `java.lang.OutOfMemoryError` from `ELKOWLOntologyLoader` when trying to deploy my custom OLS. Is there a place in the configuration where I can specify the heap size that will be used?

**Comment by balhoff** [comment]
I think I figured this out; I edited this line: https://github.com/EBISPOT/ontotools-docker/blob/e58cb66b7af172ea85a447806c453c7d3bc7f139/redeploy.sh#L85

to include `-e JAVA_OPTS=-Xmx12G -e JAVA_TOOL_OPTIONS=-Xmx12G` (wasn't sure which of those would work).

**Comment by matentzn** [comment]
It's a good point, we need to propoafade the Java env from the environment. Did Increasing the memory work for you?

**Comment by balhoff** [comment]
Yes it did! I haven't yet gone back and figured out which of those environment variables did the trick.
